config/initializers/clowder_config.rb

```diff
@@ -65,6 +65,8 @@ def self.instance
 
       # TODO: update with valid url later
       options["CLOUD_CONNECTOR_URL"] = ENV["CLOUD_CONNECTOR_URL"] 
+      options["SOURCES_URL"] = ENV["SOURCES_URL"] 
```

Comment:
@hsong-rh Is this also needed or can you make a different PR for that.

Reply:
I can add this in another PR